operator-permissions.tf

```diff
@@ -8,8 +8,6 @@ resource "kubernetes_namespace" "operator" {
 }
 
 resource "kubectl_manifest" "this" {
-  for_each   = data.kubectl_file_documents.docs.manifests
-  yaml_body  = each.value
-  apply_only = true
```

Comment:
Please don't remove `apply_only`. Some of the resources provided by AWS will update existing resources in the cluster provided by default (`eks:addon-manager` role for e.g.). When removing this module those resources would be removed completely leading to a degenerated state of the cluster.